edifact-edi/2.1/modules/ROOT/pages/index.adoc

```diff
@@ -407,6 +408,7 @@ Writer parameters control the writer operation (all are optional, with default v
 |initialMessageReference="1" |The initial Message Reference Number used for outgoing messages.
 |requestAcks="false" |Request acknowledgments for send interchanges using the Acknowledgment Request field (UNB.9).
 |testIndicator="" |Test indicator digit to be used on send interchanges (UNB.11). By default, interchanges are not sent as tests.
+|useSuppliedValues="false" |Use values from supplied data for control segment identifiers (UNB/UNZ segments). The default behavior is to always generate control numbers when writing as this allows you to instead use your chosen values. 
```

Comment:
```suggestion
|useSuppliedValues="false" |Use supplied data values for control-segment identifiers (UNB and UNZ segments). The default behavior is to not use supplied values but rather to choose your own when writing.
```
"when writing" what?

Please compare this wording to that used above and make them parallel, if not exactly the same.

Should the item above mention UNB and UNZ?

Is the generation of new values performed by the software or the user?

Should we specify "new": "generate new control numbers"?
Or "customized"?